### ISSUE TITLE
Have one person from the conda maintainers being mentored by packaging practitioners

### ISSUE THREAD

**Issue by jezdez** [conda/conda-build#4707]

**Comment by jezdez** [comment]
@kenodegard @beeankha I think you mentioned that you're interested to dive into conda-build a little, do you mind being assigned to this to make sure we're following up on it?

**Comment by jezdez** [comment]
@kenodegard I'm thinking if the conversations between @mbargull and @kenodegard could be the start of this ticket? Who else would be great at mentoring conda maintainers?

**Comment by jezdez** [comment]
@kenodegard @beeankha Do you feel like you're making progress on this?

**Comment by beeankha** [comment]
@jezdez Yes, I feel like progress is being made on this; we've been meeting regularly with @ryanskeith and will most likely be expanding the learning sessions to more stakeholders soon (we wanted to keep things small at first so the meetings wouldn't be overwhelming). 

Below are some PRs that got filed during our learning sessions, just to give you an idea of the things we're already doing to get into conda-build's internals and try to improve/clean things up:

- https://github.com/conda/conda-build/pull/4790
- https://github.com/conda/conda-build/pull/4791
- https://github.com/conda/conda-build/pull/4826

**Comment by jezdez** [comment]
Do you want to keep us up-to-date here or should we close this?

**Comment by beeankha** [comment]
I think there's enough momentum re: conda-build learning to be able to close this issue!

**Comment by ryanskeith** [comment]
I agree.  We are making slow and steady progress.

**Comment by jezdez** [comment]
Sounds great, thank you!
